dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/RunExecution.java

```diff
@@ -83,6 +91,12 @@ public String getExecutionTime() {
 
     public void setExecutionTime(String executionTime) {
         this.executionTime = executionTime;
+        // make life easier on AWS athena, also store duration in seconds
+        try {
+            this.executionTimeSeconds = Duration.parse(executionTime).getSeconds();
+        } catch (DateTimeParseException e) {
+            // ignore, expecting this to be caught by `ISO8601ExecutionTime` anyway
```

Comment:
Should you set it to `null` to be safe? I guess this method is typically only called once on the object, when it's created from the DB by Hibernate, so it will default to null anyway, so maybe it's unnecessary.

Reply:
Hmmm, I'm actually using this on the way into the webservice, I think a malformed time will just never make it in 
https://github.com/dockstore/dockstore/blob/develop/dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedMetricsTRSOpenApiIT.java#L369-L371